.github/workflows/codeql-analysis.yml

```diff
@@ -2,7 +2,7 @@ name: CodeQL Analysis
 
 on:
   push:
-    branches: "*"
+    branches: [ "**" ]
```

Comment:
>branches: [ "**" ] [](http://example.com/codeflow?start=4&length=18)

Opened this file in Visual Studio and this section was highlighted with an error message that the expected type should be an array. Also added the change recommended to ensure branches with slashes are included.

Reply:
GH accepts both ways. so, either are fine.